src/GuideAtom.tsx

```diff
@@ -1,33 +1,255 @@
+
+    ul li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.375rem;
+        height: 0.75rem;
+        width: 0.75rem;
+        margin-right: 0.5rem;
+        background-color: ${neutral[86]};
+        margin-left: -1.25rem;
+    }
+
+    b {
+        font-weight: bold;
+    }
+`;
+
+// .forceHeightAndWidth needed at the moment to override global image sizing
+// which forces images to 100%
+const imageStyling = css`
+    .forceHeightAndWidth & {
+        width: 100px;
+        height: 100px;
+    }
+    float: left;
+    margin-right: 16px;
+    margin-bottom: 6px;
+    object-fit: cover;
+    border-radius: 50%;
+    display: block;
+    border: 0px;
+`;
+
+const creditStyling = css`
+    ${textSans.xsmall()};
+    margin: 12px 0;
+    display: flex;
+    align-items: center;
+    svg {
+        width: 30px;
+        fill: ${neutral[60]};
+    }
+`;
+
+const Container = ({
+    id,
+    title,
+    children,
+    pillar,
+    expandCallback,
+}: {
+    id: string;
+    title: string;
+    pillar: string;
+    children: React.ReactNode;
+    expandCallback: () => void;
+}) => (
+    <div className={containerStyling} data-atom-id={id} data-atom-type="guide">
+        <details
+            className={detailStyling}
+            data-atom-id={id}
+            data-atom-type="guide"
+            data-snippet-type="guide"
+        >
+            <Summary
+                sectionTitle={'Quick Guide'}
+                pillar={pillar}
+                title={title}
+                expandCallback={expandCallback}
+            />
+            {children}
+        </details>
+    </div>
+);
+
+const Body = ({
+    html,
+    image,
+    credit,
+}: {
+    html: string;
+    image?: string;
+    credit?: string;
+}) => {
+    return (
+        <div>
+            {image && (
+                <span className={'forceHeightAndWidth'}>
+                    <img className={imageStyling} src={image} alt="" />
+                </span>
+            )}
+            <div
+                className={cx(
+                    bodyStyling,
+                    css`
+                        a {
+                            color: ${linkColourStyle};
+                            text-decoration: none;
+                            border-bottom: 0.0625rem solid ${neutral[86]};
+                            transition: border-color 0.15s ease-out;
+                        }
+
+                        a:hover {
+                            border-bottom: solid 0.0625rem
+                                ${linkColourStyleHover};
+                        }
+                    `,
+                )}
+                dangerouslySetInnerHTML={{
+                    __html: html,
+                }}
+            />
+            {credit && <Credit credit={credit} />}
+        </div>
+    );
+};
+
+const Credit = ({ credit }: { credit: string }) => (
+    <div className={creditStyling}>
+        <SvgInfo />
+        {credit}
+    </div>
 );
```

Comment:
This abstraction probably is just an unneeded bit of cognitive load, so worth inlining. 

```suggestion
            {credit && (<div className={creditStyling}><SvgInfo />{credit}</div>)}
        </div>
    );
};
```